kernel-commits.mk

```diff
@@ -8,6 +8,6 @@ KERNEL_COMMIT_arm64_v5.10.104_nvidia = 27012163c6a0
 KERNEL_COMMIT_arm64_v5.10.186_generic = 804663a82829
 KERNEL_COMMIT_arm64_v5.10.192_nvidia = bbf7ee8da79f
 KERNEL_COMMIT_arm64_v6.1.38_generic = 7ebde29217fc
-KERNEL_COMMIT_arm64_v6.1.106_generic = 22dd0af549fc
+KERNEL_COMMIT_arm64_v6.1.106_generic = 5639bec6ce45
```

Comment:
Just a heads up, after the merging of PR https://github.com/lf-edge/eve/pull/4380 this change will not be required anymore.

Reply:
#4380 has been merged

Reply:
Ok, I will not include this one.